rbd/errors.go

```diff
@@ -69,6 +69,9 @@ var (
 	RbdErrorImageNotOpen = ErrImageNotOpen
 	RbdErrorNotFound     = ErrNotFound
 	// revive:enable:exported
+
+	// ErrEncryptionFormat is returned when the encryption format specified is not recognized
+	ErrEncryptionFormat = errors.New("Unrecognized encryption format")
```

Comment:
I think this is now unused, as the input is enforced by the type system, and can probably be dropped.

Reply:
Not exactly. But this is captured at the librbd api layer and can be removed. Done.